build/taskctl/tasks.yaml

```diff
@@ -172,7 +190,7 @@ tasks:
     context: powershell-dotnet
     description: Run Functional Tests
     command:
-      - Invoke-Dotnet -Tests -pattern "*FunctionalTests" -path ${env:FUNCTIONAL_TESTS_RUN_DIR} -arguments "--logger 'trx' --results-directory /app/testresults --no-build"
+      - Invoke-Dotnet -Tests -pattern "*FunctionalTests" -path ${env:FUNCTIONAL_TESTS_RUN_DIR} -arguments "--logger 'trx' --results-directory /app/testresults"
```

Comment:
We want the `--no-build` here, it's been build in the build phase, no need to re-build it

Reply:
`--no-build` works for stacks-dotnet but not stacks-dotnet-cqrs. Removed to so that both work in this repo

Reply:
Hmm that sounds like we're not building properly on CQRS for the tests?

Reply:
Make sure the test projects have `<IsTestProject>true</IsTestProject>` BTW, there's a bug in the SDK